src/demo/layout/MainLayout.css

```diff
@@ -125,7 +124,12 @@ a {
 .menu-icon{
   font-size: 2rem;
 }
-@media only screen and (min-width: 375px) and (max-width: 767px) {
+
+
+@media only screen and (min-width: 325px) and (max-width: 767px) {
+  .mobile-error-handler{
+    padding: 0rem !important;
```

Comment:
`!important` not good. can you remove.

Reply:
yess but for the error modal popup that element have inline css so i added important otherwise it's picking up the css

Reply:
better to change the modal to not have inline and use class like everything else